prep-css-box-model/style.css

```diff
@@ -0,0 +1,16 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+  margin: 20px;
+  padding: 10px;
+  border width: 3px;
+  border style: double;
```

Comment:
Hi Sixx, some of your border styling is not being applied because the properties are missing hypens

```css
border-width: 3px;
border-style: double;
```